### ISSUE TITLE
questions about a new loss 

### ISSUE THREAD

**Issue by DabiaoMa** [CSTR-Edinburgh/merlin#150]
In the paper "Improving Trajectory Modelling for DNN-based Speech Synthesis by using Stacked Bottleneck Features and Minimum Generation Error Training", a new loss named 'minimum generation err'(MGE) is used as the loss function.

The authors think that this loss is more effective than that merlin is currently using. MGE takes the transformation between dynamic acoustic features(c, delta c, delta delta c) and static features (c) into consideration. 

I wonder if we could implement this loss here and how to implement this loss?
[paper.pdf](https://github.com/CSTR-Edinburgh/merlin/files/1009560/paper.pdf)
